src/messaging/dotnet/src/Server/Server/MessageRouterServer.cs

```diff
@@ -332,47 +333,47 @@ private async Task ProcessMessages(Client client, CancellationToken cancellation
                     switch (message.Type)
                     {
                         case MessageType.Connect:
-                            await HandleConnectRequest(client, (ConnectRequest)message, cancellationToken);
+                            await HandleConnectRequest(client, (ConnectRequest) message, cancellationToken);
 
                             break;
 
                         case MessageType.Subscribe:
-                            await HandleSubscribeMessage(client, (SubscribeMessage)message, cancellationToken);
+                            await HandleSubscribeMessage(client, (SubscribeMessage) message, cancellationToken);
 
                             break;
 
                         case MessageType.Unsubscribe:
-                            await HandleUnsubscribeMessage(client, (UnsubscribeMessage)message, cancellationToken);
+                            await HandleUnsubscribeMessage(client, (UnsubscribeMessage) message, cancellationToken);
 
                             break;
 
                         case MessageType.Publish:
-                            await HandlePublishMessage(client, (PublishMessage)message, cancellationToken);
+                            await HandlePublishMessage(client, (PublishMessage) message, cancellationToken);
 
                             break;
 
                         case MessageType.Invoke:
-                            await HandleInvokeRequest(client, (InvokeRequest)message, cancellationToken);
+                            await HandleInvokeRequest(client, (InvokeRequest) message, cancellationToken);
 
                             break;
 
                         case MessageType.InvokeResponse:
-                            await HandleInvokeResponse(client, (InvokeResponse)message, cancellationToken);
+                            await HandleInvokeResponse(client, (InvokeResponse) message, cancellationToken);
```

Comment:
Again, personal preference, I preferred the original style without the extra space